docs/strict-mode.md

```diff
@@ -21,6 +21,7 @@ To disable all strict mode restrictions use option `strict: false`. Some of the
     - contradictory types
     - require applicable types
   - [Strict number validation](#strict-number-validation)
+  - [Strict required](#strict-required)  
```

Comment:
I would put it as a subsection after `overlap between "properties" and "patternProperties"` - it seems related